frontend/packages/kubevirt-plugin/src/constants/url-params.ts

```diff
@@ -1,6 +1,13 @@
+import { hasV1APIVersion } from '../flags/detect-kubevirt-virtual-machines';
+
 export enum VMWizardURLParams {
   NAMESPACE = 'namespace',
   MODE = 'mode',
   INITIAL_DATA = 'initialData',
   VIEW = 'view',
 }
+
+export const VIRTUALMACHINES_BASE_URL = `kubevirt.io~${
```

Comment:
we already have a method for that.
@glekner please review

Reply:
`kubevirtReferenceForModel(VirtualMachineModel)`

Reply:
kubevirtReferenceForModel(VirtualMachineModel)